packages/compose/src/higher-order/with-global-events/listener.js

```diff
@@ -27,10 +22,10 @@ class Listener {
 	}
 
 	remove( /** @type {any} */ eventType, /** @type {any} */ instance ) {
-		this.listeners[ eventType ] = without(
-			this.listeners[ eventType ],
-			instance
-		);
+		this.listeners[ eventType ] =
+			this.listeners[ eventType ]?.filter(
+				( /** @type {any} */ listener ) => listener !== instance
+			) ?? [];
```

Comment:
Instead of `?.` and `??`, let's just return early `if ( ! this.listeners[ eventType ] )`? Then removing a non-existent listener is a complete noop that doesn't modify the object at all.

Reply:
Good call, updated in https://github.com/WordPress/gutenberg/pull/44568/commits/09ed2149c2bf651b264430ecd856f3a653eccd55